### ISSUE TITLE
Brickifying with the 'Point Cloud (Experimental)' method is broken

### ISSUE THREAD

**Issue by bblanimation** [bblanimation/bricker#63]
When attempting the 'Point Cloud' generation method (with gap and random loc/rot set to 0, if relevant), the following error was thrown:

```
Error: script failed, file: '/tmp/background_processing/brickify_in_background_template_deleteme__Sphere.py', exiting.
JOB CANCELLED: deleteme__Sphere  (returncode:155)

------ ISSUE WITH BACKGROUND PROCESSOR ------

[stderr]
Traceback (most recent call last):
  File "/Users/cgear13/Library/Application Support/Blender/2.82/scripts/addons/bricker/operators/brickify_in_background.py", line 59, in execute
    BRICKER_OT_brickify.brickify_active_frame(self.action)
  File "/Users/cgear13/Library/Application Support/Blender/2.82/scripts/addons/bricker/operators/brickify.py", line 652, in brickify_active_frame
    coll_name, _ = create_new_bricks(source_dup, parent, source_dup_details, dimensions, action, split=cm.split_model, cur_frame=None)
  File "/Users/cgear13/Library/Application Support/Blender/2.82/scripts/addons/bricker/functions/brickify_utils.py", line 311, in create_new_bricks
    v.normal.y = random_rot_angle[0]
TypeError: 'NoneType' object is not subscriptable

location: /Applications/Blender-2.82.app/Contents/Resources/2.82/scripts/modules/bpy/ops.py:201

location: /Applications/Blender-2.82.app/Contents/Resources/2.82/scripts/modules/bpy/ops.py:201
Traceback (most recent call last):
  File "/tmp/background_processing/brickify_in_background_template_deleteme__Sphere.py", line 72, in <module>
    bpy.ops.bricker.brickify_in_background(frame=frame if frame is not None else -1, action=action)
  File "/Applications/Blender-2.82.app/Contents/Resources/2.82/scripts/modules/bpy/ops.py", line 201, in __call__
    ret = op_call(self.idname_py(), None, kw)
RuntimeError: Error: Traceback (most recent call last):
  File "/Users/cgear13/Library/Application Support/Blender/2.82/scripts/addons/bricker/operators/brickify_in_background.py", line 59, in execute
    BRICKER_OT_brickify.brickify_active_frame(self.action)
  File "/Users/cgear13/Library/Application Support/Blender/2.82/scripts/addons/bricker/operators/brickify.py", line 652, in brickify_active_frame
    coll_name, _ = create_new_bricks(source_dup, parent, source_dup_details, dimensions, action, split=cm.split_model, cur_frame=None)
  File "/Users/cgear13/Library/Application Support/Blender/2.82/scripts/addons/bricker/functions/brickify_utils.py", line 311, in create_new_bricks
    v.normal.y = random_rot_angle[0]
TypeError: 'NoneType' object is not subscriptable

location: /Applications/Blender-2.82.app/Contents/Resources/2.82/scripts/modules/bpy/ops.py:201

---------------------------------------------
```

**Comment by bblanimation** [comment]
Addressed this issue in commit 1f04555ffb2b563630f0debe9e44f2e7476ada80. For an animated model that moves (e.g. moving, rotating cube), however, the model still isn't positioned correctly on 'Update Model' (tested with background processor, haven't tested locally).

**Comment by bblanimation** [comment]
Test file: 
[deleteme.blend.zip](https://github.com/bblanimation/bricker/files/4322382/deleteme.blend.zip)
